test/client/fixtures/sandbox/code-instrumentation/process-script-test.js

```diff
@@ -269,6 +269,14 @@ test('optional chaining', function () {
             src:      'var obj = null; var counter = 0; window.optionChainingResult = obj?.href[counter];',
             expected: void 0,
         },
+        {
+            src:      'var obj = null; window.optionChainingResult = obj?.href?.["name"]?.();',
+            expected: void 0,
+        },
+        {
+            src:      'var obj = { href: "123" }; window.optionChainingResult = obj?.link?.["name"]?.();',
+            expected: void 0,
+        },
```

Comment:
this tests do not work as expected =(
This code does not generates a script with a `__call$` expression. Please research these tests or write new ones to test `__call$` function implementation.